sympy/physics/control/lti.py

```diff
@@ -1144,3 +1198,164 @@ def _eval_rewrite_as_TransferFunction(self, num, den, **kwargs):
+            obj._inputs, obj._outputs = 1, len(arg)
+        else:
+            obj._inputs, obj._outputs = len(arg[0]), len(arg)
+            if not all(len(l) == obj._inputs for l in arg):
+                raise ValueError("Length of all the lists in the argument of"
+                    " TransferFunctionMatrix should be equal.")
+            if not all(isinstance(arg[row][col], TransferFunction)
+                for col in range(obj._inputs) for row in range(obj._outputs)):
+                raise TypeError("All the lists in the argument of TransferFunctionMatrix"
+                    " only support transfer functions in them.")
+
+            obj._var = arg[0][0].var
+            if not all(arg[row][col].var == obj._var
+                for col in range(obj._inputs) for row in range(obj._outputs)):
+                raise ValueError("All transfer functions should use the same complex"
+                    " variable of the Laplace transform.")
+        obj._shape = (obj._outputs, obj._inputs)
+        # No. of columns = # of inputs
+        # No. of rows = # of outputs
+        return obj
+
+    @property
+    def var(self):
+        return self._var
+
+    @property
+    def inputs(self):
+        return self._inputs
+
+    @property
+    def outputs(self):
+        return self._outputs
+
+    @property
+    def shape(self):
+        return self._shape
+
+    def __add__(self, other):
+        if isinstance(other, (TransferFunctionMatrix, Series)):
+            if isinstance(other, Series):
+                if other.is_SISO:
+                    raise ValueError("All the arguments of Series must be either"
+                        " TransferFunctionMatrix or Parallel objects.")
+            if not self.shape == other.shape:
+                raise ShapeError("Shapes of operands are not compatible for addition.")
+            if not self.var == other.var:
+                raise ValueError("All the TransferFunctionMatrix objects should use the same"
+                    " complex variable of the Laplace transform.")
+            return Parallel(self, other)
+
+        elif isinstance(other, Parallel):
+            if other.is_SISO:
+                raise ValueError("All the arguments of Parallel must be either TransferFunctionMatrix"
+                    " or Series objects.")
+            if not self.shape == other.shape:
+                raise ShapeError("Shapes of operands are not compatible for addition.")
+            if not self.var == other.var:
+                raise ValueError("All the TransferFunctionMatrix objects should use the same"
+                    " complex variable of the Laplace transform.")
+            arg_list = list(other.args)
+            return Parallel(self, *arg_list)
+
+        else:
+            raise ValueError("TransferFunctionMatrix cannot be added with {}.".
+                format(type(other)))
+
+    def __radd__(self, other):
+        return self + other
+
+    def __sub__(self, other):
```

Comment:
I think that this should dispatch to `__add__`

Reply:
Thanks! :sweat_smile: